### DISCUSSION TITLE
Git LFS 3.2.0 not installaing

### DISCUSSION THREAD

**Discussion by metaad** [git-lfs/git-lfs#5067] [Q&A]
I am getting below error when tried to install git lfs 3.2.0 even Git desktop client installed on the machine. 
![image](https://user-images.githubusercontent.com/106653425/177256530-b792e845-174b-4114-b609-6deb5dedaaa4.png)
![GIT LFS _Error](https://user-images.githubusercontent.com/106653425/177256614-039273c8-a52a-4ce6-aa6b-6174f1ed74c5.png)

Please check and suggest us.

**Comment by chrisd8088** [comment]
I think this is the same question discussed in #5031 and resolved in #5048, which will be included in the next release.
